zkevm-circuits/src/evm_circuit/util/common_gadget.rs

```diff
@@ -978,7 +1005,7 @@ impl<F: Field> CommonErrorGadget<F> {
     ) -> Self {
         cb.opcode_lookup(opcode.expr(), 1.expr());
 
-        let rw_counter_end_of_reversion = cb.query_word_unchecked(); // rw_counter_end_of_reversion just used for read lookup, therefore skip range check
+        let rw_counter_end_of_reversion = cb.query_word_unchecked::<2>(); // rw_counter_end_of_reversion just used for read lookup, therefore skip range check
```

Comment:
Note that the compiler requires me to add `::<T>` here. And I'm not sure if `::<2>` is sensible here. cc @hero78119.

Reply:
Make sense. Let me remove this N-type, and we can decide later what makes sense in future pr.

Reply:
Addressed in [2bc396f](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1435/commits/2bc396f1fabe2039378711494681c54309bb9590)